server/controllers/messagesController.js

```diff
@@ -10,12 +10,11 @@ const db = require('../models');
 const {Op} = require("sequelize");
 const Message = db.message;
 
-const createMessage = (message) => {
-
+const createMessage = (message,uid) => {
```

Comment:
Here you are passing in the request object `req` and the `uid`, so it makes more sense to refer to request object as `req` instead of `message`

Reply:
Addressed.